chain-impl-mockchain/src/vote/manager.rs

```diff
@@ -109,6 +113,18 @@ impl ProposalManager {
         cast: VoteCast,
     ) -> Result<Self, VoteError> {
         let payload = cast.into_payload();
+        match &payload {
+            Payload::Public { .. } => {}
+            Payload::Private {
+                encrypted_vote,
+                proof,
+            } => {
+                // TODO: verify private vote. Need committee public member keys
+                // if !chain_vote::verify_vote(, encrypted_vote, proof) {
+                //     return Err(VoteError::VoteVerificationError)
+                // }
```

Comment:
@eugene-babichenko Any idea how would we have the committee member public key here?